Assets/MixedRealityToolkit-Examples/Demos/Audio/Scripts/LoFiFilterSelection.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.EventDatum.Input;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces.InputSystem.Handlers;
+using Microsoft.MixedReality.Toolkit.SDK.Audio.Influencers;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    [RequireComponent(typeof(AudioLoFiEffect))]
+    public class LoFiFilterSelection : MonoBehaviour, IMixedRealityPointerHandler
+    {
+        [Tooltip("Material used when the emitter is set to Narrow Band Telephony")]
+        [SerializeField]
+        private Material NarrowBandTelephony = null;
+
+        [Tooltip("Material used when the emitter is set to AM Radio")]
+        [SerializeField]
+        private Material AmRadio = null;
+
+        [Tooltip("Material used when the emitter is set to Full Range")]
+        [SerializeField]
+        private Material FullRange = null;
+
+        [Tooltip("Material used when the emitter is set to an unknown quality")]
+        [SerializeField]
+        private Material UnknownQuality = null;
+
+        private AudioLoFiEffect loFiEffect = null;
+        // Component.renderer has been deprecated. It is safe to hide it and reuse the name.
+        private new Renderer renderer = null;
```

Comment:
let's rename it anyway so we don't get build warnings

Reply:
(I'm also not keep on using pragma to hide it)

Reply:
![image](https://user-images.githubusercontent.com/13334553/45517425-f337d280-b762-11e8-8f4c-2d28f2c5943e.png)

Reply:
what do you suggest i call it other than renderer

Reply:
`loFiSelectionRenderer` or similar I suppose. Other suggestions welcome.